lib/realtime/tenants/connect.ex

```diff
@@ -147,6 +148,7 @@ defmodule Realtime.Tenants.Connect do
 
   defp call_external_node(tenant_id, opts) do
     with tenant <- Tenants.Cache.get_tenant_by_external_id(tenant_id),
+         :ok <- tenant_suspended?(tenant),
          {:ok, node} <- Realtime.Nodes.get_node_for_tenant(tenant) do
       :erpc.call(node, __MODULE__, :connect, [tenant_id, opts], @erpc_timeout_default)
     end
```

Comment:
This should fail and buble up `:tenant_suspended` in the `RealtimeChannel` so we can log a nice error for people (and respond with it on the join) but with it like this it will explode on a no match error.